lib/dw/svelteChart.js

```diff
@@ -75,7 +75,7 @@ class Chart extends Store {
         // set a new dataset, or reset the old one if ds===true
         if (arguments.length) {
             if (ds !== true) this._dataset_cache = ds;
```

Comment:
to be honest I think we should just do:
```suggestion
            if (ds === true) ds = this._dataset_cache;
```

Reply:
yes